.gitignore

```diff
@@ -2,6 +2,7 @@ numpyro.egg-info
 __pycache__/
 .ipynb_checkpoints/
 build
+tutorials/source/examples/
```

Comment:
We don't have a tutorials directory, is this needed?

Reply:
this should be `notebooks/source/examples/` I guess.

Reply:
Thanks!